app/src/main/java/com/example/composetemplate/presentation/screens/entry_screens/login/LoginScreenEnum.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.composetemplate.presentation.screens.entry_screens.login
+
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.OffsetMapping
+import androidx.compose.ui.text.input.TransformedText
+import androidx.compose.ui.text.input.VisualTransformation
+
+enum class LoginScreenEnum {
+    EMAIL, PASSWORD, FULL_NAME, CONFIRM_PASSWORD ;
+
+    fun getPlaceHolder(): String{
+        return when(this) {
+            EMAIL -> "Email"
```

Comment:
Also here, put it in Constants file please.

Reply:
I will do it, good notice